Fix for 4k

oranke opened the pull request:
I love your awesome project. 
I added some modify for 4k monitor.

oranke committed:
add .gitignore

oranke committed:
aware 4k resolution

gtarawneh commented:
@oranke thanks for taking interest. Would you be able to make a couple of small modifications to the PR before I merge?

- Can you please delete lines instead of commenting them out or disabling them with `if false`? the current diff is difficult to read so this'll make it easier to see what changed.
- Can you please amend the commit messages to describe what was changed more specifically, particularly for commit e75079e?

oranke committed:
Add param:heroIndex.
Change neighbours's position.

oranke commented:
Sorry for my mistake. 
I'm trying to translate your code to c++. 
So I fixed the some value's position and forgot restore comments before PR what you said :)
In my game, Hero's index not always zero. therefor "SpaceSearch2" method needs additional parameter.

I fixed commits and rebased it. Thx for your review. :)
